docs: clarify map(parseInt) example with explicit radix

KeenoLee opened the pull request:
Use an arrow-function callback to explicitly forward only the string and a radix of 10 to parseInt:

// GOOD: we only forward the string (and an explicit radix)
['11','5','3'].map(n => parseInt(n, 10)); // [11, 5, 3]

KeenoLee committed:
docs: clarify map(parseInt) example with explicit radix

Use an arrow-function callback to explicitly forward only the string and a radix of 10 to parseInt:

```js
// GOOD: we only forward the string (and an explicit radix)
['11','5','3'].map(n => parseInt(n, 10)); // [11, 5, 3]

vapier commented on tsguide.html:
```diff
@@ -1561,8 +1561,8 @@ <h4 id="functions-as-callbacks" class="numbered">Prefer passing arrow functions
-<pre><code class="language-ts good">// GOOD: Arguments are explicitly passed to the callback
-const numbers = ['11', '5', '3'].map((n) =&gt; parseInt(n));
+<pre><code class="language-ts good">// GOOD: we only forward the string (and an explicit radix)
+const numbers = ['11', '5', '3'].map((n) =&gt; parseInt(n,10));
```
missing space after the `,`

KeenoLee replied:
Thank you for your review. I just updated the code based on the comment

vapier commented on tsguide.html:
```diff
@@ -1561,8 +1561,8 @@ <h4 id="functions-as-callbacks" class="numbered">Prefer passing arrow functions
 
-<pre><code class="language-ts good">// GOOD: Arguments are explicitly passed to the callback
-const numbers = ['11', '5', '3'].map((n) =&gt; parseInt(n));
+<pre><code class="language-ts good">// GOOD: we only forward the string (and an explicit radix)
```
I don't think the comment needs to spell things out like this.  the previous comment is fine.

KeenoLee replied:
No problem, I reverted to the previous comment

KeenoLee committed:
Merge branch 'google:gh-pages' into gh-pages

KeenoLee committed:
docs: simplify GOOD callback comment

Shorten the “GOOD” example comment to its original concise form, per maintainer feedback:

vapier commented:
i checked internally and it looks like we also ran into this issue.  unfortunately they switched to using an internal API, so i've asked them to tweak it again to address your feedback while being usable externally.

KeenoLee committed:
Merge branch 'gh-pages' into gh-pages

KeenoLee committed:
Merge branch 'gh-pages' into gh-pages
